### DISCUSSION TITLE
Different dynamic color in dark mode

### DISCUSSION THREAD

**Discussion by NikunjK8** [fast4x/RiMusic#1339] [Q&A]
Shouldn't the dynamic color be Yellow just like in light mode? It turns green-ish for some reason. Other songs are fine.

![IMG_20240326_162420](https://github.com/fast4x/RiMusic/assets/99175083/0d012028-9b5a-46c7-91c9-6fe25be4f404)
![IMG_20240326_162353](https://github.com/fast4x/RiMusic/assets/99175083/8b9f8338-84b1-4aa4-90a9-8fab6339802b)
![IMG_20240326_162124](https://github.com/fast4x/RiMusic/assets/99175083/96a8899b-1394-42e4-830d-d5f5a7b64ba0)

**Comment by fast4x** [comment]
Try disable gradient

**Reply by NikunjK8** [reply]
Still the same

**Comment by fast4x** [comment]
Song link please

**Comment by fast4x** [comment]
for me work fine
![Screenshot_2024-04-01-21-42-32-289_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/1484476/72979c4f-2b89-4ef0-b20c-e9bfe88f82b2)

**Reply by NikunjK8** [reply]
It works fine in light mode. Try switching to dark mode.

**Reply by fast4x** [reply]
Dark

![Screenshot_2024-04-01-21-47-14-144_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/1484476/98798154-73e1-472b-b731-6e67f342cd94)

Pitch Black

![Screenshot_2024-04-01-21-47-25-609_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/1484476/367862b2-99cf-4ce9-bdbd-5fe6a3fe83f3)

**Reply by NikunjK8** [reply]
Why does it change color instead of keeping it Yellow?

**Reply by OrangeZXZ** [reply]
I think in dark mode it should still keep the gold color but darker instead of taking another color maybe a issue to make

**Reply by NikunjK8** [reply]
Yes, that's what I'm talking about. Just want to make if it's actually worth making the issue

**Reply by fast4x** [reply]
Dark is dark...

**Reply by fast4x** [reply]
It’s okay for me so... there’s more priority to do...

**Reply by NikunjK8** [reply]
That's Fine. It was happening with one song so i asked. No big deal
